Sources/ArcGISToolkit/Resources/FloorFilterSiteFacilityCell.xib

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="17701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="17703"/>
+        <capability name="Image references" minToolsVersion="12.0"/>
+        <capability name="System colors in document resources" minToolsVersion="11.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" restorationIdentifier="FloorFilterSiteFacilityCell" insetsLayoutMarginsFromSafeArea="NO" selectionStyle="blue" hidesAccessoryWhenEditing="NO" indentationLevel="1" indentationWidth="0.0" reuseIdentifier="FloorFilterSiteFacilityCell" rowHeight="30" id="9Jm-xW-tsw" userLabel="FloorFilterSiteFacilityCell" customClass="SiteFacilityTableViewCell" customModule="ArcGISToolkit">
+            <rect key="frame" x="0.0" y="0.0" width="277" height="45"/>
+            <autoresizingMask key="autoresizingMask"/>
+            <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" userInteractionEnabled="NO" contentMode="center" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="9Jm-xW-tsw" id="sS3-2K-4kh">
+                <rect key="frame" x="0.0" y="0.0" width="277" height="45"/>
+                <autoresizingMask key="autoresizingMask"/>
+                <subviews>
+                    <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="chevron.right" catalog="system" translatesAutoresizingMaskIntoConstraints="NO" id="St7-Qf-W9T" userLabel="siteFacilityRightChevron">
+                        <rect key="frame" x="236.5" y="13" width="12.5" height="16.5"/>
+                        <color key="tintColor" systemColor="systemGray3Color"/>
+                    </imageView>
```

Comment:
Since this is a table view cell, please use a [disclosure indicator](https://developer.apple.com/documentation/uikit/uitableviewcell/accessorytype/disclosureindicator) instead of a custom image.